packages/select/README.md

```diff
@@ -0,0 +1,128 @@
+# Select
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/select.svg)
+
+#### [View on Storybook](https://mongodb.github.io/leafygreen-ui/?path=/story/select--default)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/select
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/select
+```
+
+## Example
+
+```js
+import Select, { Option, OptionGroup, Size } from '@leafygreen-ui/select';
+
+<Select
+  label="Label"
+  description="Description"
+  placeholder="Placeholder"
+  name="Name"
+  size={Size.Normal}
```

Comment:
I think we should move in the direction of Size.Default. Most of our components follow this convention anyway (with the exception of Button and Pipeline) and it feels more correct